.gitmodules

```diff
@@ -4,6 +4,9 @@
 [submodule "libs/crunch"]
 	path = libs/crunch
 	url = https://github.com/DaemonEngine/crunch.git
+[submodule "libs/freetype"]
+	path = libs/freetype
```

Comment:
Thoughts on committing the source code directly instead of a submodule?

We could have done that with Googletest too...

Reply:
I prefer submodules, committing may be good for things like some dead single-file include or so, and even in such situation it depends… Submodules make it easier to check for updates, to sync with upstream, and to commit that change.